imap_processing/idex/idex_packet_parser.py

```diff
@@ -133,35 +161,335 @@ def __init__(self, header_packet):
         self.Target_High_bits = ""
         self.Ion_Grid_bits = ""
 
-        # Log the rest of the header
-        self._log_packet_info(header_packet)
+    def _get_trigger_dicts(self, packet):
+        """
+        This function creates a large dictionary of values from the FPGA header
+        that need to be captured into the CDF file.  They are lumped together because
+        they share similar attributes.
+
+        Notes about the variables are set here, acting as comments and will also be
+        placed into the CDF in the VAR_NOTES attribute.
+
+        Parameters
+        ----------
+            packet : The IDEX FPGA Header Packet
+
+        Returns
+        -------
+            dict
+                A dictionary of (CDF variable name : value) pairs
+            dict
+                A dictionary of (CDF variable name : variable notes) pairs
+        """
+        trigger_dict = {}
+        trigger_notes_dict = {}
+
+        # Get Event Number
+        trigger_dict["event_number"] = packet.data["IDX__TXHDREVTNUM"].raw_value
+        trigger_notes_dict[
+            "event_number"
+        ] = "The unique number assigned to the impact by the FPGA"
+        # TOF High Trigger Info 1
+        tofh_trigger_info_1 = packet.data["IDX__TXHDRHGTRIGCTRL1"].raw_value
+        trigger_dict["tof_high_trigger_level"] = tofh_trigger_info_1 & self.TEN_BIT_MASK
+        trigger_notes_dict[
+            "tof_high_trigger_level"
+        ] = "Trigger level for the TOF High Channel"
+        trigger_dict["tof_high_trigger_num_max_1_2"] = (
+            tofh_trigger_info_1 >> 10 & self.ELEVEN_BIT_MASK
+        )
+        trigger_notes_dict[
+            "tof_high_trigger_num_max_1_2"
+        ] = """Maximum number of samples between pulse 1 and 2 for TOF High double
+               pulse triggering"""
+        trigger_dict["tof_high_trigger_num_min_1_2"] = (
+            tofh_trigger_info_1 >> 21 & self.ELEVEN_BIT_MASK
+        )
+        trigger_notes_dict[
+            "tof_high_trigger_num_min_1_2"
+        ] = """Minimum number of samples between pulse 1 and 2 for TOF High double
```

Comment:
I see now. `.shortDescription` would have nice in this file. It's very informative.